docs/Flint.schema.json

```diff
@@ -0,0 +1,681 @@
+                  },
+                  "function": {
+                    "type": "string"
+                  },
+                  "references": {
+                    "type": "string"
+                  },
+                  "dutyHolder": {
+                    "type": "string"
+                  },
+                  "claimant": {
+                    "type": "string"
+                  },
+                  "create": {
+                    "type": "string"
+                  },
+                  "terminate": {
+                    "type": "string"
+                  },
+                  "enforce": {
+                    "type": "string"
+                  },
+                  "explanation": {
+                    "type": "string"
+                  },
+                  "actor": {
+                    "type": "string"
+                  },
+                  "action": {
+                    "type": "string"
+                  },
+                  "eObject": {
+                    "type": "string"
+                  },
+                  "recipient": {
+                    "type": "string"
+                  },
+                  "precondition": {
+                    "type": "string"
+                  },
+                  "language": {
+                    "type": "string"
+                  },
+                  "sources": {
+                    "type": "string"
+                  },
+                  "text": {
+                    "type": "string"
+                  },
+                  "findLineInSource": {
+                    "type": "string"
+                  },
+                  "validFrom": {
+                    "type": "string"
+                  },
+                  "validTo": {
+                    "type": "string"
+                  },
+                  "tags": {
+                    "type": "string"
+                  },
+                  "version": {
+                    "type": "string"
+                  },
+                  "verified": {
+                    "type": "string"
+                  },
+                  "validated": {
+                    "type": "string"
+                  },
+                  "publicized": {
+                    "type": "string"
+                  },
+                  "selectVersion": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "languageName",
+                  "name",
+                  "acts",
+                  "facts",
+                  "duties",
+                  "function",
+                  "references",
+                  "dutyHolder",
+                  "claimant",
+                  "create",
+                  "terminate",
+                  "enforce",
+                  "explanation",
+                  "actor",
+                  "action",
+                  "eObject",
+                  "recipient",
+                  "precondition",
+                  "language",
```

Comment:
how does the language property differ from languageName?

Reply:
`languageName` -> Name of the specific language used
`language` -> translation for the word 'language' e.g. 'taal'

Reply:
@cblazo please resolve the conversation if the answer is satisfactory